civihr_employee_portal/views/templates/views-view--my-details.tpl.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+  // This is the view for printing contact details in My Details page
```

Comment:
I'm not clear why the other views were in the $footer section to begin with

Reply:
@mickadoo because those are embedded views in the footer. It is a common way to relate content in Drupal.